homeworks/Andrii.Morozov_AndriiMorozov88/5-rxjs/script_rxjs.js

```diff
@@ -0,0 +1,24 @@
+const { fromEvent } = window.rxjs;
+const {
+    map, pairwise, throttleTime,
+} = window.rxjs.operators;
+const logo = document.querySelector('[data-header]');
+const scroll$ = fromEvent(window, 'scroll');
+scroll$.pipe(
+    map(() => window.scrollY),
+    throttleTime(200),
+    pairwise(),
+    map((pairDifference) => pairDifference[1] - pairDifference[0]),
```

Comment:
```suggestion
    map(([previous, current]) => current - previous),
```
Іноді краще використовувати деструкторизацію в масивах